### DISCUSSION TITLE
Is it possible to configure bullmq to use a single key as a list for the jobs in redis?

### DISCUSSION THREAD

**Discussion by luiscabus** [taskforcesh/bullmq#2626] [Q&A]
In case I am using redis for many things, the jobs keys makes things very polluted, requiring additional software/logic to read redis.

In bee-queue, for example, only a few keys are used, even if I enqueue thousand things, with one of them containing all the jobs to be picked up. Laravel queue also does it that way, using one hash only to store all the jobs.

But bullmq will use one key for each job I push, is it possible to configure it to "group" the jobs in one key/hash?

**Comment by manast** [comment]
The answer will be the same as I wrote for your question in the Bull repo: https://github.com/OptimalBits/bull/discussions/2753#discussioncomment-9901615

**Reply by luiscabus** [reply]
Oh I just wanted to make sure I've asked in the correct repo! :)
